mobile-app/app/components/skeletonLoaders/BrowseAuctionsLoader.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import ContentLoader, { Circle, IContentLoaderProps, Rect } from 'react-content-loader/native'
+import { useThemeContext } from '@shared-contexts/ThemeProvider'
+import { tailwind } from '@tailwind'
+import { ThemedView } from '../themed'
+
+type BrowseAuctionsLoaderProps = JSX.IntrinsicAttributes & IContentLoaderProps & { children?: React.ReactNode }
+
+export function BrowseAuctionsLoader (props: BrowseAuctionsLoaderProps): JSX.Element {
+  const { isLight } = useThemeContext()
+  return (
+    <ThemedView
+      dark={tailwind('bg-gray-800 border border-gray-700')}
+      light={tailwind('bg-white border border-gray-200')}
+      style={tailwind('px-2 mt-2 items-center justify-center')}
```

Comment:
```suggestion
      style={tailwind('px-2 mb-2 rounded items-center justify-center')}
```
to better match your batch card UI

Reply:
Updating